lms/envs/common.py

```diff
@@ -321,7 +321,15 @@
 
     'DISABLE_AUDIT_CERTIFICATES': False,  # Toggle to disable audit certificates
 
-    # for acceptance and load testing
+    # .. toggle_name: FEATURES['AUTOMATIC_AUTH_FOR_TESTING']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set to True to perform acceptance and load test. Auto auth view is responsible for load
+    #   testing and is controlled by this feature flag.
```

Comment:
@jramnai could you please also mention that this disables the CSRF middleware?

Reply:
@eLRuLL, I couldn't find any documentation or code base to disable CSRF middleware but I found [some code](https://github.com/edx/edx-platform/blob/6f2d54d53b3b8f52ff44489f879aa354a1fc8f21/openedx/core/djangoapps/cache_toolbox/middleware.py#L133#L134) that disables session verification when this feature flag is set to True. I have updated the toogle_description with it.

Reply:
so I got that impression from the PR you linked in the setting description: https://github.com/edx/edx-platform/pull/417/files#diff-a16fd1c6cf75bdebdf97a2633b533d39e43975c6e98af58787ed6af1282eca71R161

now that I see it, it was referring actually to `AUTOMATIC_AUTH_FOR_LOAD_TESTING` and not `AUTOMATIC_AUTH_FOR_TESTING`, was it maybe renamed or it's a different one? That information could also be useful, maybe understanding on which release it was renamed (if it was)

Reply:
@eLRuLL, initially this flag was added as `AUTOMATIC_AUTH_FOR_LOAD_TESTING` and then renamed with `AUTOMATIC_AUTH_FOR_TESTING` in [this commit](https://github.com/edx/edx-platform/pull/608/commits/e78edf8ef9f0eb1d4a0799e75e602dabaedfc116) in 2013. I don't think so that we should add this information as it will lead to confusion for newbies. And version information is also not available with the rename commit. Let me know your thoughts.

Reply:
@jramnai yep totally agree, looks good now